### DISCUSSION TITLE
How do I install vcpkg for system-wide using in linux

### DISCUSSION THREAD

**Discussion by soroshsabz** [microsoft/vcpkg#29558] [Q&A]
ITNOA

I have a question, I want to install vcpkg system-wide for using all of my users in system (including root user) but I do not found any command in `vcpkg --help` or in `bootstrap.sh` for this purpose

thanks

**Comment by autoantwort** [comment]
Please don't do that. Only one user at a time will be able to use vcpkg.

**Reply by soroshsabz** [reply]
Why? I want to use vcpkg like any other tools in system (for example cmake, git, g++ and etc.) why I have to use relative path everyday? and all user in the same system have to install specific vcpkg? it is very unefficient

**Reply by Neumann-A** [reply]
because vcpkg is intended to be project bound and not system bound.

**Reply by soroshsabz** [reply]
many ports is very huge, and downloading again and again is wasting time and volume of internet

**Reply by autoantwort** [reply]
You can vote for https://github.com/microsoft/vcpkg/issues/22790

**Reply by Neumann-A** [reply]
> and downloading again and again is wasting time and volume of internet

This can be avoided by setting up vcpkg to share a downloads folder, asset and binary cache. All which is currently possible and i use.

**Reply by soroshsabz** [reply]
@Neumann-A  I think for windows and Linux, vcpkg have to provide user friendly method for these requirements

And if I understand correctly for using `vcpkg` binary system wide, I have to copy binary to `/usr/local/bin` and set environmental variables? vcpkg does not provide any native method for this?
